.github/workflows/triviy.yml

```diff
@@ -0,0 +1,86 @@
+name: Trivy scan
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '46 8 * * 0'
+  push:
+    branches: ["master"]
+  pull_request:
+
+# Declare default permissions as nothing.
+permissions: {}
+
+jobs:
+  trivy-scan:
+    name: Trivy scan
+    runs-on: ubuntu-22.04
+    strategy:
+      matrix:
+        branch: [master, release/2.6]
```

Comment:
How does this interact with https://github.com/daos-stack/daos/blob/61b178522d7e13df791563c7887a1eb6d533df9c/.github/workflows/triviy.yml#L7-L8

Are we going to be scanning both `master` and `release/2.6` on every push to `master`?

Are we going to backport this to `release/2.6` and if so, the same question applies there about scanning `master` when PRs land to `release/2.6`?

Reply:
Initially, I tried to build logic for all branches in one place, but it seems to be better to have a separate workflow for each branch (master + release/*)

Reply:
Indeed.  The attraction of using a matrix here is understandable but in this context I don't think it's desirable.

Moreover, considering what will happen in the future, when `master` is branched to `release/2.8` where both branches will now contain this workflow, doing a backport for `release/2.6` makes the most sense and provides the best consistency with how things will work going forward.